libs/velo-external-db-core/test/drivers/schema_matchers.ts

```diff
@@ -27,59 +36,34 @@ export const schemaHeadersListFor = (collections: any) => toHaveSchemas(collecti
 
 export const schemasWithReadOnlyCapabilitiesFor = (collections: any) => toHaveSchemas(collections, collectionToHaveReadOnlyCapability)
 
-
-const toHaveCollection = ( collections: any[], functionOnEachCollection: any, ...args: any ) =>  expect.objectContaining({
-    collection: collections.map((c: any) => functionOnEachCollection(c, args))
-})
-
-export const queryOperatorsFor = (fieldType: string): string[] => {
-    switch (fieldType) {
-        case 'text':
-        case 'url':
-            return [eq, ne, string_contains, string_begins, string_ends, include, gt, gte, lt, lte]
-        case 'number':
-            return [eq, ne, gt, gte, lt, lte, include]
-        case 'boolean':
-            return [eq]
-        case 'image':
-            return []
-        case 'object':
-            return [eq, ne]
-        case 'datetime':
-            return [eq, ne, gt, gte, lt, lte]
-    
-        default:
-            throw new Error(`${fieldType} - Unsupported field type`)
-    }
-
-}
-
-export const fieldInNewWixFormat = (field: any) => expect.objectContaining({
+export const fieldInNewWixFormat = (field: ResponseField) => expect.objectContaining({
     key: field.field,
     type: fieldTypeToWixDataEnum(field.type),
-    encrypted: false,
     capabilities: expect.objectContaining({
-        sortable: expect.any(Boolean),
-        queryOperators: queriesToWixDataQueryOperators(queryOperatorsFor(field.type))
+        sortable: field.capabilities!.sortable,
+        queryOperators: expect.arrayContaining(field.capabilities!.columnQueryOperators.map(c => queryOperatorsToWixDataQueryOperators(c)))
     })
-
 })
 
-export const capabilitiesInNewWixFormat = () => expect.objectContaining({
-    dataOperations: expect.any(Array),
-    fieldTypes: expect.any(Array),
-    collectionOperations: expect.any(Array),
+export const fieldsInWixFormat = (fields: ResponseField[]) => expect.arrayContaining(fields.map(f => fieldInNewWixFormat(f)))
+
+export const capabilitiesInWixFormat = (collectionsCapabilities: CollectionCapabilities) => expect.objectContaining({
```

Comment:
rename to matcher name

Reply:
What's the problem with the naming? it's a matcher for a capabilities object in a "Wix" format

Reply:
I feel it's should be adjective (`toBe.../ toHave../ haveCapabilities/ includesNini`)